app/javascript/routes/Rooms.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import axios from "axios"
+import { Row, Col, Container, Spinner } from "react-bootstrap";
+import RoomCard from "../components/RoomCard";
+import {useQuery} from "react-query";
+
+export default function Rooms() {
+
+  const { isLoading, error, data: rooms, isFetching } = useQuery("getRooms", () =>
+    axios.get('/api/v1/rooms.json', {
```

Comment:
Synchronizing with the new changes introduced in https://github.com/bigbluebutton/greenlight/pull/3321 should further simplify this.